src/rdkafka_topic.h

```diff
@@ -231,6 +241,22 @@ rd_kafka_topic_t *rd_kafka_topic_find0_fl(const char *func,
         rd_kafka_topic_find_fl(__FUNCTION__, __LINE__, rk, topic, do_lock)
 #define rd_kafka_topic_find0(rk, topic)                                        \
         rd_kafka_topic_find0_fl(__FUNCTION__, __LINE__, rk, topic)
+/**
+ * Same semantics as ..find() but takes a Uuid instead.
+ */
+rd_kafka_topic_t *rd_kafka_topic_find_by_topic_id(rd_kafka_t *rk, rd_kafka_Uuid_t topic_id){
+        rd_kafka_topic_t *rkt;
+
+        rd_kafka_rdlock(rk);
+        TAILQ_FOREACH(rkt, &rk->rk_topics, rkt_link) {
+                if (!rd_kafka_Uuid_cmp(rkt->rkt_id, topic_id)){
+                        rd_kafka_topic_keep(rkt);
+                        break;
+                }
+        }
+        rd_kafka_rdunlock(rk);
+        return rkt;
+}
```

Comment:
This is the cause of the compilation error, remove the implementation and leave only the declaration here.
Also, remove the comment as it's defined in the implementation.

```suggestion
rd_kafka_topic_t *rd_kafka_topic_find_by_topic_id(rd_kafka_t *rk, rd_kafka_Uuid_t topic_id);
```